atomic_reactor/plugins/post_pulp_sync.py

```diff
@@ -249,7 +249,7 @@ def run(self):
         # manifests from Koji metadata if they are not present
         # (i.e. if Pulp does not have v2 schema 2 support).
         self.log.info("fetching repository content")
-        content = pulp.listRepo(list(repos.values()), content=True)
+        content = pulp.listRepos(list(repos.values()), content=True)
         manifest_refs = list(content.keys())
```

Comment:
@twaugh, looks like we missed one. `content` is a list of dicts in this case. This is currently raising an exception:
```
2017-09-12 18:35:16,885 platform:- - atomic_reactor.plugin - DEBUG - Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugin.py", line 242, in run
    plugin_response = plugin_instance.run()
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugins/post_pulp_sync.py", line 253, in run
    manifest_refs = list(content.keys())
AttributeError: 'list' object has no attribute 'keys'
```

Reply:
See https://github.com/projectatomic/atomic-reactor/pull/827